### PR TITLE
OBSDOCS-132: Update collector pod docs

### PR DESCRIPTION
**Pull request by abrennan89**
Version(s):
4.11+

Issue:
https://issues.redhat.com/browse/OBSDOCS-132

Link to docs preview:
https://67610--docspreview.netlify.app/openshift-enterprise/latest/logging/log_collection_forwarding/cluster-logging-collector#cluster-logging-collector-pod-location_cluster-logging-collector

QE review:
- [x] QE has approved this change.

### CODE REVIEW

File: modules/cluster-logging-collector-pod-location.adoc [hunk 1]
```diff
@@ -6,43 +6,24 @@
 [id="cluster-logging-collector-pod-location_{context}"]
 = Viewing logging collector pods
 
-You can view the Fluentd logging collector pods and the corresponding nodes that they are running on. The Fluentd logging collector pods run only in the `openshift-logging` project.
+You can view the logging collector pods and the corresponding nodes that they are running on. The logging collector pods run only in the `openshift-logging` project.
```
Comment (anpingli) [inline modules/cluster-logging-collector-pod-location.adoc:9]: that is not correct now.

**Review by jcantrill — COMMENTED** [review]
/lgtm

File: modules/cluster-logging-collector-pod-location.adoc (hunk 1 above)
Comment (abrennan89) [reply modules/cluster-logging-collector-pod-location.adoc]: Updated to remove the name of the project and the outdated sentence.

**Review by jeana-redhat — COMMENTED** [review]
LGTM

/remove-label peer-review-in-progress
/remove-label peer-review-needed
/label peer-review-done

**Pull request merged**